blog/_posts/2016-09-21-Buffering.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Buffering
+layout: post
+created_at: 2016-09-21
+permalink: blog/2016-09-19-Buffering
+current: blog
+author: Thea Amanda Kupler and Marie-Luise Kochsiek
+twitter: @loadtocode
+categories:
+- student-posts
+---
+## <span class="color-red">...so the Rails Girls Summer of Code will come to an end!</span>
+
+Two months ago we blogged about our start - about us being [loaded to code](/blog/2016-07-21-start-of-loadtocode). Now, having some time passed, which seems to go faster day by day, there is this constant feeling of loading, loading knowledge to code or rather buffering - buffering all the knowledge we received those last months. We not only received great coding teaching but also valuable business knowledge. We reached out to the amazing SoundCloud-People surrounding us and mentioned that we are a little bit lost in our learning freedom and not having a structure set from top-down. We saw the problem, reached out and found solutions - creating our own structure. BOTTOM(S)-UP.
+
+![LoadtoCode KANBAN](/img/blog/2016/2016-09-21-KANBAN.gif) <font color="grey"><small><i>READY, STEADY, PUSH, DONE: Our KANBAN-Board in movement!</i></small></font><br>
```

Comment:
add image credits to all of your images, as explained [here](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#images).